test/FormLabelSpec.tsx

```diff
@@ -0,0 +1,169 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import FormLabel from '../src/FormLabel';
+import FormGroup from '../src/FormGroup';
+
+import { shouldWarn } from './helpers';
+
+describe('<FormLabel>', () => {
+  it('should render correctly', () => {
+    const { getByTestId } = render(
+      <FormLabel
+        id="foo"
+        htmlFor="bar"
+        className="my-control"
+        data-testid="test-id"
+      />,
+    );
+
+    const element = getByTestId('test-id');
+    element.tagName.toLowerCase().should.equal('label');
+    element.classList.length.should.equal(2);
+    element.classList.contains('form-label').should.be.true;
+    element.classList.contains('my-control').should.be.true;
+    element.id.should.equal('foo');
+    element.hasAttribute('for');
```

Comment:
We don't need this line since the next line asserts the value of the attribute

Reply:
Fixed in the commit https://github.com/react-bootstrap/react-bootstrap/pull/6224/commits/023f9b3e2cffbb507001f992d76bb9cfb4243310